packages/front-end/components/Experiment/PValueColumn.tsx

```diff
@@ -59,10 +59,13 @@ const PValueColumn: FC<{
     suspiciousChange,
     belowMinChange,
   });
+
+  const expected: number = stats.expected ?? 0;
   const expectedDirection = metric.inverse
-    ? stats.expected < 0
-    : stats.expected > 0;
-  const statSig = stats.pValue < pValueThreshold;
+    ? expected < 0
+    : expected > 0;
+  const pValue: number = stats.pValue ?? 1;
+  const statSig = pValue < pValueThreshold;
```

Comment:
This addresses the problem, however I'm wondering if it might be better to exit early / render nothing if the above values are missing? Ultimately will defer to your opinion on this but curious to hear what you think.

Reply:
Yeah I was wondering the same thing, but decided to mostly ape the logic in `ChanceToWinColumn`. I can test both with my broken MongoDB and see what seems better.